tests/flexmock_test.py

```diff
@@ -1087,6 +1299,46 @@ def get_stuff(self):
         flexmock(user).should_call("get_stuff").and_raise(FlexmockError)
         user.get_stuff()
 
+    def test_flexmock_should_reraise_exception_in_spy(self):
+        class RaisesException:
+            def get_stuff(self):
+                raise RuntimeError("foo")
+
+        instance = RaisesException()
```

Comment:
Should we have those tests with class and static methods also ?

Reply:
I extended the test cases: https://github.com/flexmock/flexmock/pull/61/commits/93f69a6239142247a2e06e45a072463a486f1571